build/.vsts-PR-internal-check-healthcare-shared-components-azureBuild-pipeline.yml

```diff
@@ -0,0 +1,81 @@
+name:  pr$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:-r)
+
+parameters:
+- name: BuildConfiguration
+  default: release
+
+variables:
+- name:  vmImage
+  value: windows-latest 
+
+jobs:
+
+- job: Phase_1
+  displayName: Phase 1
+  cancelTimeoutInMinutes: 1
+  pool:
+    vmImage: $(vmImage)
+
+  steps:
+  - checkout: self
+
+  - task: UseDotNet@2
+    displayName: Use .NET Core sdk 
+    inputs:
+      useGlobalJson: true
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet restore
+    inputs:
+      command: restore
+      projects: '**/*.sln'
+      selectOrConfig: config
+      nugetConfigPath: nuget.config
+
+  - task: DotNetCoreCLI@1
+    name: ''
+    displayName: dotnet build
+    inputs:
+      projects: '**/*.sln'
+      arguments: --configuration ${{ parameters.BuildConfiguration }} --version-suffix $(build.buildnumber)
```

Comment:
Wait -- we're not running any tests in this pipeline! lol

Reply:
@wsugarman I did get your comments. Could you please clarify? Thanks!

Reply:
We should be running unit tests in our pipeline, right? We should be using dotnet test to ensure all of our unit tests are passing

Reply:
The unit tests are run as part of a separate pipeline (Github action technically): https://github.com/microsoft/healthcare-shared-components/runs/4144107071?check_suite_focus=true